backend/prompt_studio/prompt_studio_core/exceptions.py

```diff
@@ -23,7 +33,7 @@ class IndexingAPIError(APIException):
     default_detail = "Error while indexing file"
 
 
-class AnswerFetchError(APIException):
+class AnswerFetchError(CustomAPIException):
```

Comment:
```suggestion
class AnswerFetchError(APIException):
```
Any reason for using this `CustomAPIException` class? Anyway what it does very similar to `APIException`

Reply:
Removed already. Thanks for pointing it out.